src/region_manager/workspace.rs

```diff
@@ -0,0 +1,58 @@
+use std::cell::{Ref, RefCell, RefMut};
+use std::hash::{Hash, Hasher};
+use std::rc::Rc;
+
+use smithay::desktop;
+
+#[derive(derivative::Derivative)]
+#[derivative(Debug)]
+pub struct WorkspaceInner {
+    #[derivative(Debug = "ignore")]
+    space: desktop::Space,
```

Comment:
What is the reason for this ignore?

Reply:
Hmm version of smithay with I used locally when doing it didn't have Debug on Space as far I remember. Now I see it has it now, so yes, this can be removed, no reason to use that.